### ISSUE TITLE
Filter for disabling the notice box?

### ISSUE THREAD

**Issue by khromov** [TGMPA/TGM-Plugin-Activation#190]
Is it possible to add a filter for disabling the notice box? We're working on a multisite installation with a parent theme that uses TGM and we don't want it to show up on every admins profiles.

We have hooked into get_user_metadata to override the default setting value globally, but it would be better if there was a native filter available.

Here's the code we are using now:

```php
add_filter('get_user_metadata', function($val, $object_id, $meta_key, $single)
{
    if($meta_key === 'tgmpa_dismissed_notice')
        return true;
    else
        return null;
 
}, 10, 4);
```

More info:
http://snippets.khromov.se/disable-tgm-plugin-activation-nag-screen/

**Comment by jrfnl** [comment]
@khromov PR #247 makes it so that on multisite, the notices are only shown in the network admin. Hope this helps a bit. Feel free to +1 it if you approve of the PR.

Oh and to completely disable the admin notices, just change `'has_notices'` in your config array to false, no filter needed (has been available since v2.1.0):
`$config['has_notices'] = false;`

**Comment by khromov** [comment]
Nice! :+1:
